Sources/Sentry/SentryClient.m

```diff
@@ -509,6 +507,35 @@ - (SentryEvent *_Nullable)callEventProcessors:(SentryEvent *)event
     return newEvent;
 }
 
+- (void)setSdk:(SentryEvent *)event
```

Comment:
l: If the `integrations` key may be replaced by the value on the event, you might as well check if `__sentry_sdk_integrations` exists before doing  all the string replace stuff. 

I know. Im being very pick here. 😉

Reply:
Good catch. I think we can even remove `__sentry_sdk_integrations` as they aren't used anywhere else. I didn't want to do this in this PR and want' to open another PR for that.